apps/studio/stores/RootStore.ts

```diff
@@ -35,16 +31,8 @@ export class RootStore implements IRootStore {
    * This method will also trigger project detail loading when it's not available
    */
   setProject(project: Project) {
-    if (this.selectedProjectRef === project.ref) return
-
-    // reset ui projectRef in case of switching projects
-    // this will show the loading screen instead of showing the previous project
-    this.ui.setProjectRef(undefined)
```

Comment:
This line is unnecessary since we only call the `setProject()` method with an actual project.